test/e2e/angular-app/src/app/login.component.ts

```diff
@@ -0,0 +1,54 @@
+// src/app/login.component.ts
+
+import { Component, OnInit } from '@angular/core';
+import { Router, NavigationStart} from '@angular/router';
+
+import { OktaAuthService } from '@okta/okta-angular';
+import * as OktaSignIn from '@okta/okta-signin-widget';
```

Comment:
Hmm. I thought we fixed this with `2.5`. Can we not import the widget via:

```typescript
import OktaSignIn from '@okta/okta-signin-widget';
```

Reply:
You can do that in our create-react-app example, because it's a bit looser with how it handles ES6 modules. TypeScript is strict, so you must import like:
`import * as OktaSignIn from '@okta/okta-signin-widget';`

We fixed the need to reference dist:
`import * as OktaSignIn from '@okta/okta-signin-widget/dist/js/okta-sign-in.min.js';`